### PR TITLE
add github actions

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -49,7 +49,7 @@ def mavenBuild(jdk, cmdline) {
           mavenOpts: mavenOpts,
           mavenLocalRepo: localRepo) {
     // Some common Maven command line + provided command line
-    sh "mvn -V -B -DfailIfNoTests=false -Dmaven.test.failure.ignore=true $cmdline"
+    sh "mvn -V -B -DfailIfNoTests=false -fae $cmdline"
```
Comment (joakime) [inline Jenkinsfile:52]: perhaps we should make the 2 CI builds equivalent?

Have both be ..

```
mvn -V -B -e -DfailIfNoTests=false -fae clean verify
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
```
Comment (joakime) [inline .github/dependabot.yml:2]: There's also a docker ecosystem here.

Consider adding ...
```
  - package-ecosystem: "docker"
    # Look for a `Dockerfile` in the `root` directory
    directory: "/"
    schedule:
      interval: "weekly"
```

We might need to tweak this `directory` setting later.

**Review by joakime — CHANGES_REQUESTED** [review]

**Comment by lachlan-roberts** [comment]
@olamy this is failing the tests
> dependabot.yml Failing after 1s — Your .github/dependabot.yml contained invalid details

Maybe remove Travis CI in this PR? Although it does seem to be passing now.

**Comment by lachlan-roberts** [comment]
I guess we're scrapping the use of that `hack-my-builds.sh` script from the docker guys then. The GithubCI is just doing the same test as we do on the Jenkins build.

It looks like the docker-library images have moved to using GithubCI as well, maybe we could do something similar to what they do there.  https://github.com/docker-library/openjdk/blob/master/.github/workflows/ci.yml

**Comment by lachlan-roberts** [comment]
@olamy this is failing eca check, I don't think you signed latest commit

**Comment by olamy** [comment]
@lachlan-roberts  arghh I did that using merge option from GH UI bad idea :) I will fix that

**Pull request merged**